cmd/flowrunner/testdata/flows/all_actions.json

```diff
@@ -9,7 +9,9 @@
             "localization": {
                 "spa": {
                     "ac110f56-a66c-4462-921c-b2c6d1c6dadb": {
-                        "text": ["Hola @contact.name, ¿estás listo?"]
+                        "text": [
+                            "Hola @contact.name, ¿estás listo?"
+                        ]
```

Comment:
Looks like maybe golang 1.10.1 changed some default JSON formatting so there are some whitespace diffs in here.

Reply:
This isn't in a file that our tests are writing, no? 

I was about to ask you how you formatted these all in vscode :) Either way I'm fine with these changes because I have an JSON formatting extension in vscode to keep JSON files consistent and this is how it formats them

Reply:
Oh hah, you are right, I must have opened this in vscode and it formatted them. Well, I guess that's fine too.